lib/bing.js

```diff
@@ -66,7 +66,7 @@ var Bing = function (options) {
     + vertical
     + (vertical == 'spellcheck' ? "?text=" : "?q=") + query
     + (opts.top ? "&count=" + opts.top : "")
-    + (opts.offset ? "&offset=" + opts.skip : "")
+    + (opts.skip ? "&offset=" + opts.skip : "")
```

Comment:
This would break the current version. I'd add the support for both options in that case.
So `(opts.offset || opts.skip) ? '&offset=' + (opts.offset || opts.skip) : ''` would do.